include/fmt/format.h

```diff
@@ -3484,9 +3484,18 @@ template <typename Char> struct udl_arg {
   }
 };
 
+// Converts string literals to basic_string_view.
 template <typename Char, size_t N>
-FMT_CONSTEXPR basic_string_view<Char> literal_to_view(const Char (&s)[N]) {
-  return {s, N - 1};
+FMT_CONSTEXPR basic_string_view<Char> fmt_string_to_view(const Char (&s)[N]) {
+  // Remove trailing null character if needed. Won't be present if this is used with
+  // raw character array (i.e. not defined as a string).
```

Comment:
Please apply `clang-format` to your changes.

Reply:
Attempting to clang-format this file is resulting in a massive number of changes for me. Can you please clarify how my changes are not in conformance?

Reply:
I'm seeing a ton of reformatting when using clang-format on this file. I'm going to leave for now and this can be addressed separately.